src/main/resources/templates/c_plus_plus/test/testUtils/junit/TestCase.py

```
@@ -0,0 +1,75 @@
+            result: Et.Element = Et.SubElement(case, self.result.value)
+            result.set("message", self.message)
+            result.text = self.genErrFailureMessage()
+
+        if self.stdout:
+            stdout: Et.Element = Et.SubElement(case, "system-out")
+            stdout.text = shortenText(self.stdout, maxCharsPerOutput) + "\n"
+        if self.stderr:
+            stderr: Et.Element = Et.SubElement(case, "system-err")
+            stderr.text = shortenText(self.stderr, maxCharsPerOutput) + "\n"
+
+    def genErrFailureMessage(self, maxChars: int = 5000) -> str:
```

Ruff

Missing type annotation for `self` in method

(ANN101)